src/model.cpp

```diff
@@ -546,6 +546,10 @@ std::vector<std::string> Model::getObservableIds() const {
     return std::vector<std::string>();
 }
 
+bool Model::hasQuadraticLLH() const {
+    return true;
```

Comment:
It's a virtual method that can be overridden by subclasses in case they don't implement a gaussian llh. Setting this to `true` by default ensure backwards compatibility.